indico/modules/events/sessions/templates/display/session_display.html

```diff
@@ -0,0 +1,115 @@
+        {%- endif %}
+        <div class="group">
+            <a href="{{ url_for('.export_session_timetable', sess) }}" class="i-button icon-file-pdf"></a>
+            {% if sess.start_dt -%}
+                <a id="exportIcal{{ sess.id }}" href="#"
+                   class="i-button icon-calendar exportIcal"
+                   data-id="{{ sess.id }}"></a>
+            {%- endif %}
+        </div>
+        {% if sess.start_dt -%}
+            {% with item=sess, ics_url=url_for('.export_ics', sess) -%}
+                {% include 'events/sessions/display/session_ical_export.html' %}
+            {%- endwith %}
+        {%- endif %}
+    </div>
+{%- endblock %}
+
+{% block content -%}
+    <div id="session-display">
+        <div class="infogrid condensed">
+            <div class="infoline">
+                <i class="icon-time"></i>
+                <div class="text">
+                    <div>
+                        <span>
+                            {% if sess.start_dt -%}
+                                {{ sess.start_dt|format_datetime }}
+                            {%- else -%}
+                                {% trans %}Session not scheduled yet.{% endtrans %}
+                            {%- endif %}
+                        </span>
+                    </div>
+                    <div>
+                        <span>
+                            {% if sess.end_dt -%}
+                                {{ sess.end_dt|format_datetime }}
+                            {%- endif %}
+                        </span>
+                    </div>
+                </div>
+            </div>
+        </div>
+        {% if sess.blocks -%}
+            <div class="infogrid condensed">
+                <div class="infoline">
+                    <div class="text">
+                        <h2>{% trans %}Conveners{% endtrans %}</h2>
+                        {% for block in sess.blocks -%}
+                            <span><strong>{{ block.title }}</strong></span>
+                            <ul>
+                                {% for convener in block.person_links -%}
+                                    <li class="icon-user">
+                                        <a href="mailto:{{ convener.person.email }}">{{ convener.person.full_name }}</a>
+                                        ({{ convener.person.affiliation }})
+                                    </li>
+                                {%- else -%}
+                                    <li>{% trans %}There are no conveners in this block{% endtrans %}</li>
+                                {%- endfor %}
+                            </ul>
+                        {%- endfor %}
+                    </div>
+                </div>
+            </div>
+        {%- endif %}
+
+        <h2>{% trans %}Session contributions{% endtrans %}</h2>
+        <div id="session-contrib-list" class="contribution-list">
+            {{ render_contribution_list(event, session_contribs) }}
```

Comment:
that contribution list repeats the session - but on this page that's pretty redundant